src/fragments/lib/project-setup/flutter/create-application/10_createProject.mdx

```diff
@@ -1,34 +1,7 @@
-**Open Android Studio.**  Select **+ Start a new Flutter project**
-
-![](/images/lib/getting-started/flutter/set-up-android-studio-welcome.png)
-
- In **Select a Project Template**, select **Flutter Application**. Press **Next**.
-
-Next, configure your project:
-
-1. Enter *MyAmplifyApp* in the **Name** field
-2. Make sure your Flutter SDK path is set correctly to where it is installed on your machine 
-3. Press **Next**. On the next screen, press **Finish**. 
-
-  ![](/images/lib/getting-started/flutter/set-up-android-studio-configure-your-project.png)
-
-Android Studio will open your project with a tab opened to *main.dart*
-
-import flutter0 from "/src/fragments/lib/project-setup/native_common/prereq/flutter_null_safety.mdx";
-
-<Fragments fragments={{flutter: flutter0}} />
-
-If you use CocoaPods, modify your Podfile to target iOS platform 11.0 or higher.  Within your project open `ios/Podfile` and change the second line to be `platform :ios, '11.0'.
+Create a new project using Flutter CLI:
 
 ```bash
-sudo gem install cocoapods
-
-# make sure you are in the ios folder
-pod init
+flutter create <your_app_name>
 ```
 
-In XCode, select your project in the Project Navigator and select the "Info" tab.  Set the Deployment Target to at least 11.0.
-
-  ![](/images/lib/getting-started/flutter/set-up-xcode-deployment-target.png)
-
```

Comment:
This bit is still needed, but maybe we can put it below with the iOS stuff and change 11->13